docs/generators/java-camel.md

```diff
@@ -56,6 +56,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
 |enumUnknownDefaultCase|If the server adds new enum cases, that are unknown by an old spec/client, the client will fail to parse the network response.With this option enabled, each enum will have a new case, 'unknown_default_open_api', so that when the server sends an enum case that is not known by the client/spec, they can safely fallback to this case.|<dl><dt>**false**</dt><dd>No changes to the enum's are made, this is the default option.</dd><dt>**true**</dt><dd>With this option enabled, each enum will have a new case, 'unknown_default_open_api', so that when the enum case sent by the server is not known by the client/spec, can safely be decoded to this case.</dd></dl>|false|
 |fullJavaUtil|whether to use fully qualified name for classes under java.util. This option only works for Java API client| |false|
+|generatedConstructorWithRequiredArgs|Whether to generate constructors with required args for models| |false|
```

Comment:
Hmm... I wonder if the default should be `true`.

Your use case with Lombok is interesting, but I would think by default developers should populate all required params when constructing a DTO/POJO - so I think this should default to true and if you want to use custom Lombok annotations, you can set to false?

What do you think?

Reply:
@welshm from that perspective you are correct, but I'm thinking more from backward compatible perspective. I'm totally fine to change this to opposite value

Reply:
@welshm I've changed default value to `true` and I've marked this change as "Breaking with fallback" so we should be fine from release notes perspective